internal/server/repository_worker.go

```diff
@@ -292,7 +292,7 @@ func (r *Repository) UpsertWorkerStatus(ctx context.Context, worker *Worker, opt
 				// "description" since we want description changes for PKI-based
 				// workers to come via API only. We can't really guard on this
 				// in the DB so we need to be sure to not include it here.
-				n, err := w.Update(ctx, workerClone, []string{"address"}, nil)
+				n, err := w.Update(ctx, workerClone, []string{"address", "ReleaseVersion"}, nil)
```

Comment:
It seems odd that here it would be `ReleaseVersion` and below `release_version`. I realize gorm probably ends up making it the same but feels like it should be consistent.

Reply:
It is odd but Update fieldMasks and the create on conflict db.SetColumns accept different column name formats. Update will not accept release_version and will error and inversely SetColumns would err on ReleaseVersion. 
It looks like SetColumns expect the actual column name and Update converts field mask paths into column names in go.dbw